docs/users.md

```diff
@@ -2,12 +2,76 @@
 
 HTTP Verb | Endpoint                   | Description                | Link
 ----------|----------------------------|----------------------------|---------------------------
+GET       | `/users/find?google_id={google_id}`         | Retrieve a user by their Google ID.         | [Link](#find-user)
 GET       | `/users/{user_id}`         | Get a single user.         | [Link](#get-user)
 POST      | `/users`                   | Create a new user.         | [Link](#create-new-user)
 PATCH     | `/users/{user_id}`         | Update an existing user.   | [Link](#update-user)
 
 ---
 
+## Find User
+
+Returns a user based on their Google id to facilitate OAuth.
+
+```
+GET /users/find?google_id={google_id}
+```
+
+### Parameters
+
+Name       | Data Type    | In    | Required/Optional | Description
+-----------|--------------|-------|-------------------|------------
+`google_id`  | Integer      | Path  | Required          | The Google ID of the user.
```

Comment:
`google_id` in `Query Parameters` instead of `Path`

Reply:
Updated accordingly